<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateUser</class>
 <widget class="QDialog" name="CreateUser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>381</width>
    <height>167</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>To-Do List New User</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLineEdit" name="newUsername">
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Username</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="newPassword">
       <property name="echoMode">
        <enum>QLineEdit::EchoMode::Password</enum>
       </property>
       <property name="placeholderText">
        <string>Password</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="newPasswordConfirm">
       <property name="echoMode">
        <enum>QLineEdit::EchoMode::Password</enum>
       </property>
       <property name="placeholderText">
        <string>Confirm password</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="createNewAccountButton">
       <property name="text">
        <string>Create Account</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
